docs/built-in-bpa-rules.md

```diff
@@ -19,6 +30,11 @@ Function Apps should require HTTPS to ensure connections are made to the expecte
 
 **Recommendation**: To [use HTTPS to ensure server/service authentication and protect data in transit from network layer eavesdropping attacks](https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-bindings#enforce-https), in the [Microsoft.Web/Sites resource properties](https://docs.microsoft.com/en-us/azure/templates/microsoft.web/sites?tabs=json#siteproperties-object), add (or update) the *httpsOnly* property, setting its value to `true`.
 
+## Only secure connections to your Azure Cache for Redis should be enabled
+Enable only connections via SSL to Redis Cache. Use of secure connections ensures authentication between the server and the service and protects data in transit from network layer attacks such as man-in-the-middle, eavesdropping, and session-hijacking.
+
+**Recommendation**: To [enable only connections via SSL to Redis Cache](https://docs.microsoft.com/en-us/security/benchmark/azure/baselines/azure-cache-for-redis-security-baseline?toc=/azure/azure-cache-for-redis/TOC.json#44-encrypt-all-sensitive-information-in-transit), in the [Microsoft.Cache/Redis resource properties](https://docs.microsoft.com/en-us/azure/templates/microsoft.cache/redis?tabs=json#rediscreateproperties-object), add (or update) the *enableNonSslPort* property, setting its value to `false`.
```

Comment:
We should update the guidance slightly, since we're taking the default value of 'false' into account.  They can change the value from true to false, or just delete the property.

Reply:
Thoughts on this wording?

Recommendation: To enable only connections via SSL to Redis Cache, in the Microsoft.Cache/Redis resource properties, add (or update) the enableNonSslPort property, setting its value to false or remove the property if the Microsoft.Cache/redis Template API version you are using has false as it's default value.

Reply:
I don't think we need to worry about different API versions in the recommendation.  It's extremely unlikely a new version will be published that changes the default to 'true', making it less secure than previous versions.  However, if *previous* versions have a different default value, we should change the rule back to enforce false.

Currently, since the rule only requires the property isn't 'true', the recommendation shouldn't suggest adding the property, since it won't flag to begin with if the property doesn't already exist.  Another suggestion:

Recommendation: To enable only connections via SSL to Redis Cache, in the Microsoft.Cache/Redis resource properties, update the value of the *enableNonSslPort* property from `true` to `false`, or remove the property from the template.

Reply:
My concern is if previous versions had a different default value. I'll update the recommendation, but we should test out previous versions.

Reply:
I just made test deployments. Default value for versions "2020-06-01" and later have false as the default value.